deployments/examples/opencloud_full/config/keycloak/docker-entrypoint-override.sh

```diff
@@ -2,7 +2,9 @@
 printenv
 # replace openCloud domain in keycloak realm import
 mkdir /opt/keycloak/data/import
-sed -e "s/cloud.opencloud.test/${OC_DOMAIN}/g" /opt/keycloak/data/import-dist/opencloud-realm.json > /opt/keycloak/data/import/opencloud-realm.json
+sed -e "s/cloud.opencloud.test/${OC_DOMAIN}/g" \
```

Comment:
What is the gain of that change?
Before your change, it was already a valid config, because `cloud.opencloud.test` is used on all testing and developer machines.
 ${OC_DOMAIN} is not always populated if you do not run the entrypoint script.

So i would like to keep it like it was.